spec/Section 6 -- Execution.md

```diff
@@ -302,7 +304,8 @@ ExecuteSubscriptionEvent(subscription, schema, variableValues, initialValue):
 - Let {data} be the result of running {ExecuteSelectionSet(selectionSet,
   subscriptionType, initialValue, variableValues)} _normally_ (allowing
   parallelization).
-- Let {errors} be any _field errors_ produced while executing the selection set.
+- Let {errors} be the list of all _field error_ raised while executing the
```

Comment:
Plural?

Reply:
For this to be a glossary referenced term it need to remain in the singular, hence "list of field error" vs "field errors"